packages/ember-testing/lib/helpers/current_url.js

```diff
@@ -1,3 +1,18 @@
+/**
```

Comment:
Can you move this comment down to just above the `currentURL` function export below?